back-office/shop-parameters/general/maintenance.md

```diff
@@ -26,4 +26,8 @@ Clicking 'Save', users are expected to stay on this page and see a success notif
 
 ## Multistore behavior
 
-[TO BE COMPLETED]
+In a specific shop context or in a group context, the options listed above should have [checkboxes](https://github.com/PrestaShop/PrestaShop/issues/19359) 
+
+In an all shops context, the options listed above should have [specific settings drop-down](https://github.com/PrestaShop/PrestaShop/issues/19312)
```

Comment:
```suggestion
In an all shops context, the options listed above should have [specific settings drop-down](https://github.com/PrestaShop/PrestaShop/issues/19314)
```

Shouldn't it be the link of the "Multistore - Shop Parameters > General > Maintenance - Add specific settings dropdown" issue instead of the Webservice one ?

Reply:
Yes, nice catch, I made a bad copy/past :+1: